src/proof_system/widget/range/mod.rs

```diff
@@ -4,17 +4,10 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+#[cfg(feature = "alloc")]
 mod proverkey;
 mod verifierkey;
```

Comment:
Add a new line before `mod verifierkey` so it's clearer that `feature alloc` is applied only to `proverkey`.